test/correctness/tester.cpp

```diff
@@ -27,8 +27,8 @@ template <typename T, typename U>
 Tester<T,U>::Tester(int argc, char *argv[], const bool silent,
                     const std::string &name, const std::vector<std::string> &options):
     help_("Options given/available:\n"),
-    platform_(Platform(GetArgument(argc, argv, help_, kArgPlatform, size_t{0}))),
-    device_(Device(platform_, GetArgument(argc, argv, help_, kArgDevice, size_t{0}))),
+    platform_(Platform(GetArgument(argc, argv, help_, kArgPlatform, ConvertArgument(getenv("CLBLAST_PLATFORM"), size_t{0})))),
+    device_(Device(platform_, GetArgument(argc, argv, help_, kArgDevice, ConvertArgument(getenv("CLBLAST_DEVICE"), size_t{0})))),
```

Comment:
Where is `getenv` defined? Perhaps better to make it `std::getenv` and include `<cstdlib>` to be on the safe side (http://en.cppreference.com/w/cpp/utility/program/getenv).